Sources/Basic/JSON.swift

```diff
@@ -92,18 +92,18 @@ extension JSON: Equatable {
 
 extension JSON {
     /// Encode a JSON item into a string of bytes.
-    public func toBytes(prettyPrint: Bool = false) -> ByteString {
+    public func toBytes(prettyPrint: Bool = false, sortingKeysBy: (String, String) -> Bool = { $0 < $1 }) -> ByteString {
```

Comment:
Nit: for consistency, could you use the internal parameter name here too (as in, `sortingKeysBy areInIncreasingOrder: (String, String) -> Bool`)?